packages/pymedphys_deliverydata/src/pymedphys_deliverydata/logfile/trf.py

```diff
@@ -41,38 +42,37 @@ def delivery_data_from_logfile(logfile_path):
     return delivery_data_from_pandas(logfile_dataframe)
 
 
-def delivery_data_from_pandas(logfile_dataframe):
-    raw_monitor_units = logfile_dataframe[
-        'Step Dose/Actual Value (Mu)'].values.tolist()
+def delivery_data_from_pandas(logfile_dataframe) -> DeliveryDataBase:
```

Comment:
See point made above about need for separate function...

Reply:
In this case I disagree more strongly. This pandas dataframe format is very, very specific to the trf logfile format.